src/Main.elm

```diff
@@ -67,20 +88,43 @@ init =
 -- Update
 
 
+toUpper : String -> String
+toUpper str =
+    String.toUpper (String.left 1 str) ++ String.dropLeft 1 str
```

Comment:
Totally stole this from another package on the internet that only did this, but wasn't updated for our elm version.